<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pidefecha</class>
 <widget class="QDialog" name="pidefecha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Petici√≥n de fecha</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>INTRODUZCA FECHA</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="firstDayOfWeek">
      <enum>Qt::Monday</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Concepto:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="conceptolineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Documento:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="documentolineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aceptarpushButton">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>calendarWidget</tabstop>
  <tabstop>conceptolineEdit</tabstop>
  <tabstop>documentolineEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>aceptarpushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>aceptarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>pidefecha</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>pidefecha</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
